src/pages/admin/reports/components/funds-expenditures-section/components/common/edit-funds-expenditures-category-modal/EditFundsExpendituresCategoryModal.module.scss

```
@@ -0,0 +1,209 @@
+    align-items: center;
+    justify-content: space-between;
+    border: 1px solid c.$grey-700;
+    border-radius: 8px;
+    background-color: c.$white;
+}
+
+.input {
+    width: 100%;
+}
+
+.panel :global(.char-limit-input) {
```

Stylelint (17.10.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)